.template/addons/github/.github/workflows/deploy_aws_fargate.yml.tt

```diff
@@ -0,0 +1,50 @@
+name: Deploy to AWS Fargate
+
+on:
+  workflow_run:
+    workflows:
+      - Test
+    branches:
+      - main
+    types:
+      - completed
+  workflow_dispatch:
+
+env:
+  DOCKER_REGISTRY_HOST: ${{ secrets.DOCKER_REGISTRY_HOST }}
+  DOCKER_IMAGE: ${{ secrets.ECR_REPO }}
+  ECS_CLUSTER: ${{ secrets.ECS_CLUSTER }}
+  ECS_SERVICE: ${{ secrets.ECS_SERVICE }}
+  BRANCH_TAG: latest
```

Comment:
We usually use `nimblehq/branch-tag-action@v1.2` to get `BRANCH_TAG`, should we do the same to make it consistent?

Reply:
I aware of that but for the main/master branch, that `branch-tag-action` action always returns the tag as `latest` so I don't think we need to spend more time running that action. What do you think?

Reply:
🤔 It's unnecessary if we only run it on main/master branch. But will we run this workflow on another branch manually? If no, it's ok to set the fixed env.
 I saw you set `workflow_dispatch` event, then it can be triggered manually. In my projects, I sometimes trigger the deployment manually on staginng to test some changes before deploying to production.

Reply:
that's a good point. But I'm concern that by using the same `latest` tag for staging (and production), we only have one image on ECR, new image with the same `latest` tag will override the previous one and it saves the storing cost. But if we use the `branch-tag-action` action to generate the image tag & then deploying any random branch to staging, new image with a different tag will be pushed & left to ECR. What do you think?

Reply:
@hoangmirs I removed this workflow to keep this PR for the automated code review workflow only. The workflow to deploy to ECS will be included in another PR